src/library/scala/collection/TraversableView.scala

```diff
@@ -25,6 +25,9 @@ trait TraversableView[+A, +Coll] extends TraversableViewLike[A, Coll, Traversabl
  *  `TraversableView`s work. Its definitions are generally not accessed directly by clients.
  */
 object TraversableView {
+  private[this] val reusedNoBuilder = new NoBuilder[Any]
```

Comment:
What do we gain from caching this value? You can't do anything useful with `NoBuilder`. Is it ever created but then not used?

Reply:
I was just following the same pattern - reverting the changes to this file

Reply:
done